app/models/manageiq/providers/azure/cloud_manager/scanning/job.rb

```diff
@@ -167,4 +171,9 @@ def delete_snapshot_and_reset_snapshot_mor(log_verb)
     end
   end
 
+  def create_scan_args
+    super.tap do |scan_args|
+      scan_args["snapshot"] = {"name" => context[:snapshot_mor]}
```

Comment:
TODO the snapshot "name" is only used by MiqAzureVm in the case of a "blob" snapshot.  The snapshot_mor returned by the azure create_snapshot method does appear to be the name but I need to confirm this

Reply:
Okay confirmed the old code used the `context[:snapshot_mor]` as the `scan_args["snapshot"]["name"]` : https://github.com/ManageIQ/manageiq/pull/19607/files#diff-e8144513ccccfdc45e012a21875a9b40017058f6b00d0e4a4908326c245af0c9L202